packages/neuron-ui/src/components/Settings/index.tsx

```diff
@@ -120,9 +120,11 @@ const Settings = () => {
   return (
     <PageContainer head={t('navbar.settings')}>
       <div className={styles.container}>
-        {items.map(([title, ItemCmp]) => (
+        {items.map(([title, marginTop, ItemCmp]) => (
```

Comment:
items can be defined as `[string, string, React.FC<any>][]` then we can remove `as string`